helix-view/src/theme.rs

```diff
@@ -56,24 +58,28 @@ impl Loader {
             return Ok(self.base16_default());
         }
 
-        let theme = self.load_theme(name, name, false).map(Theme::from)?;
+        let mut visited_paths = HashSet::new();
```

Comment:
Hi, I'm trying to merge this into #5636. What's the point of this `HashSet` introduction?

Reply:
Nevermind, sorry. Didn't see the new `load_theme` logic.